relay-quotas/src/redis.rs

```diff
@@ -129,24 +129,14 @@ impl<'a> RedisQuota<'a> {
 
     /// Returns the key of the quota.
     pub fn key(&self) -> String {
-        if matches!(self.quota.scope, QuotaScope::Global) {
```

Comment:
no longer necessary because we construct the redis key separately for global scoped quotas.

Reply:
left comment about prefix below